packages/sample-addon/package.json

```diff
@@ -33,7 +33,7 @@
     "ember-cli": "*",
     "ember-cli-dependency-checker": "*",
     "ember-cli-eslint": "*",
-    "ember-cli-htmlbars-inline-precompile": "*",
+    "ember-cli-htmlbars-inline-precompile": "~2.1.0",
```

Comment:
All of these sample package's package.json's should use `*` dependencies for everything. The real versions are controlled by `packages/ember-auto-import/package.json` which should be the only one that uses real versions.

Reply:
The problem is that it brings in `ember-cli-htmlbars-inline-precompile` 3.0. Which removes compatibility with babel 6.  This was my first approach and it just broke every test that used babel 6.

Reply:
https://travis-ci.org/ef4/ember-auto-import/builds/590091845

This is the build where you can see the breakage it causes. The problem is that this dependency being brought in is too new, that they are too old.

Reply:
Okay so I did end up reverting MOST of the  `ember-cli-htmlbars-inline-precompile` to `*`.

Except for the packages that were failing because of the newer API: that is `sample-direct` and `sample-addon`.  In those I loosened the package floating to `<3.0.0` in order for it to float as much as possible.